c_src/eleveldb.cc

```diff
@@ -353,14 +357,15 @@ ERL_NIF_TERM parse_open_option(ErlNifEnv* env, ERL_NIF_TERM item, leveldb::Optio
         }
         else if (option[0] == eleveldb::ATOM_COMPRESSION)
         {
-            if (option[1] == eleveldb::ATOM_TRUE)
-            {
+            if (option[1] == eleveldb::ATOM_ON || option[1] == eleveldb::ATOM_TRUE
+                || option[1] == eleveldb::ATOM_SNAPPY )
                 opts.compression = leveldb::kSnappyCompression;
-            }
```

Comment:
I ALWAYS use curly braces around block statements, even if they are one-liners.  That future proofs you from stupidity, and it doesn't make the code run any slower.  'twas the rule at Iona, and I always live by it.